apps/api/src/app/auth/auth.controller.ts

```diff
@@ -183,6 +184,32 @@ export class AuthController {
 
     return res.redirect("/");
   }
+  @Post("resendVerification")
+  async resendVerificationMail(@Request() req: ExpressRequest) {
+    const userCookie = this.usersService.getUserInfo(req);
+
+    if (userCookie && await this.usersService.user({ id: userCookie.id })
```

Comment:
The result from `this.usersService.user({ id: userCookie.id })` is used twice - this can be condensed into a declaration before the conditional.